src/babashka/curl.clj

```diff
@@ -74,13 +74,7 @@
                  (case method
                    :head ["--head"]
                    ["--request" (-> method name str/upper-case)]))
-        headers (:headers opts)
-        headers (loop [headers* (transient [])
-                       kvs (seq headers)]
-                  (if kvs
-                    (let [[k v] (first kvs)]
-                      (recur (reduce conj! headers* ["-H" (str k ": " v)]) (next kvs)))
-                    (persistent! headers*)))
+        headers (into [] (mapcat (fn [[k v]] ["-H" (str (name k) ": " v)])) (:headers opts))
```

Comment:
Should we also allow namespaced keywords? Co-incidentally I was just looking at the latest commit/PR in clj-commons/clj-yaml.

Reply:
What translation scheme do you have in mind?

Reply:
I was just wondering if there are headers like `Foo/Bar` allows and used in HTTP?

Reply:
`(.substring (str k) 1)` is the way they are converted in cheshire

Reply:
I see the PR now, `(subs (str k) 1)`. But if I'm [reading correctly](https://stackoverflow.com/q/47687379/1688998) slashes aren't allowed in HTTP header keys

Reply:
I checked clj-http and they also seem to take the `name`:

https://github.com/dakrone/clj-http/blob/1c751431a3a8d38a795a70609a60cee24ad62757/src/clj_http/headers.clj#L63

So this PR looks good to me.

Reply:
The only reasonable thing to do with the namespace that comes to my mind is to join it with a dash
```clj
(str (when (instance? clojure.lang.Named k) (when-some [n (namespace k)] (str n "-"))) (name k))
```

That way someone can use e.g. `:access-control/allow-headers`, `:access-control/max-age` and such.

Reply:
Yeah, I wonder why that function in clj-http (link above) doesn't do this. But this might also be a little bit too much magic.

Reply:
Well, if someone will *really* need this, they can be linked to this conversation with a working snippet ready for another PR ;) Thanks for the quick resolve!